HotTips/TipOfTheDayWindow.xaml.cs

```diff
@@ -245,6 +270,70 @@ private void GroupNameCheckBox_Checked(object sender, RoutedEventArgs e)
         {
             _tipHistoryManager?.MarkTipGroupAsIncluded(GroupNameLabel.Content.ToString());
         }
+
+        private void LikeButton_Click(object sender, RoutedEventArgs e)
+        {
+            if (isLiked)
+            {
+                PopulateDislikeFilledImage();
```

Comment:
I think this should change to PopulateDislikeImage (not filled) because they are clicking like on something that's already liked so set both to unfilled.